styles/main.scss

```diff
@@ -452,21 +453,87 @@ header {
   order: 2;
 }
 
-.intro-avatar .image {
+.intro-avatar .image-avatar {
+  position: relative;
   width: 90px;
   height: 90px;
   border-radius: 50%;
   overflow: hidden;
   margin: 0 auto 16px;
 }
 
-.intro-avatar .image img {
+.intro-avatar .image-avatar:after {
```

Comment:
yea. I add the PR with refactor of Intro component to mui. I think it will be easier  for you  to check this refactor if it will be like another PR. https://github.com/1alexvash/AlexCodeBlog/pull/82